ci/test/singularity

```diff
@@ -51,4 +53,13 @@ action "custom action" {
 EOF
 
 popper run
-popper run --quiet
\ No newline at end of file
+popper run --quiet
+popper run --debug
+popper run --dry-run
```

Comment:
I missed this in my review:

  * Instead of running with all distinct flags, we can run only with `popper --debug`.
  * then we extend the `ci/test/dry-run` test so that we add singularity actions to the workflow defined there
  * create a `ci/test/quiet` that compares with and without `--quiet`, and where the workflow defined there will contain singularity actions